AudibleApi/Authentication/LoginResultRunner.cs

```diff
@@ -145,74 +119,65 @@ private static async Task<HttpResponseMessage> makeRequestAsync(Authenticate aut
 			var response = await authenticate.LoginClient.SendAsync(request);
 
 			#region debug: enumerate post-call cookies
-			if (Serilog.Log.Logger.IsDebugEnabled())
+			try
 			{
-				try
-				{
-					Serilog.Log.Logger.Debug("Cookies after request. {@DebugInfo}", authenticate.LoginClient.CookieJar.Debug_GetCookies(authenticate.Locale.LoginUri()));
-				}
-				catch (Exception ex)
-				{
-					Serilog.Log.Error(ex, "post-call cookies debug failure");
-				}
+				Serilog.Log.Logger.Debug("Cookies after request. {@DebugInfo}", authenticate.LoginClient.CookieJar.Debug_GetCookies(authenticate.Locale.LoginUri()));
+			}
+			catch (Exception ex)
+			{
+				Serilog.Log.Error(ex, "post-call cookies debug failure");
 			}
 			#endregion
 
 			#region debug: request/response details
-			if (Serilog.Log.Logger.IsDebugEnabled())
+			try
 			{
-				try
-				{
-					var requestContentLength = "null";
-					if (content is not null)
-						requestContentLength = (await content.ReadAsStringAsync()).Length.ToString();
-
-					var responseContentLength = "null";
-					if (response?.Content is not null)
-						responseContentLength = (await response.Content.ReadAsStringAsync()).Length.ToString();
-
-					Serilog.Log.Logger.Debug("Request/Response details. {@DebugInfo}", new {
-						requestUri_fromResponse = response.RequestMessage.RequestUri.AbsoluteUri,
-						requestHeaders = response.RequestMessage.Headers,
-						requestContentLength,
-						responseCode = $"{response.StatusCode}={(int)response.StatusCode}",
-						responseHeaders = response.Headers,
-						responseContentLength
-					});
-				}
-				catch (Exception ex)
-				{
-					Serilog.Log.Error(ex, "request/response details debug failure");
-				}
+				var requestContentLength = "null";
+				if (content is not null)
+					requestContentLength = (await content.ReadAsStringAsync()).Length.ToString();
+
+				var responseContentLength = "null";
+				if (response?.Content is not null)
+					responseContentLength = (await response.Content.ReadAsStringAsync()).Length.ToString();
+
+				Serilog.Log.Logger.Debug("Request/Response details. {@DebugInfo}", new {
+					requestUri_fromResponse = response.RequestMessage.RequestUri.AbsoluteUri,
+					requestHeaders = response.RequestMessage.Headers,
+					requestContentLength,
+					responseCode = $"{response.StatusCode}={(int)response.StatusCode}",
+					responseHeaders = response.Headers,
+					responseContentLength
+				});
+			}
+			catch (Exception ex)
+			{
+				Serilog.Log.Error(ex, "request/response details debug failure");
 			}
 			#endregion
 
 			#region VERBOSE debugging only: save page content
-			if (Serilog.Log.Logger.IsVerboseEnabled())
```

Comment:
Why was this check removed?

Reply:
I actually reverted this change but forgot to push it. It's done now.